tiberius/src/types/mod.rs

```diff
@@ -643,7 +643,8 @@ impl<'a, S: FromColumnData<'a> + 'a> FromColumnData<'a> for Option<S> {
 
 from_column_data!(
     // integers are auto-castable on receiving
-    bool:       ColumnData::Bit(val) => val;
+    bool:       ColumnData::Bit(val) => val,
+                ColumnData::I8(val) => val != 0;
```

Comment:
use an explicit cast for that: `SELECT CAST(column as bit)`
Allowing to read a column that's an int as bool, opens us up for hard to debug errors,
when a developer does a index or column based typo, he won't get an error but
duck-typing that will make finding the error on his part harder, than just showing an error message.

Reply:
Then, what you thing about making ColumnData public and adding (similar to get() and try_get()) one more function "get_data()" that will return reference to actual ColumnData enum ? Then I can do casting on my side. 

(Chanign SQL is a solution, but in my case it is not one or few places, but about thousandth. We have crazy idea to rewrite some big project with more then 5 million lines of code in Delphi to Rust).

Reply:
Making ColumnData public prevents internal changes of representation, that's why I'd like to avoid that.

Is the reason that doing `try_get(...).map(|x| x!= 0)` or even let x = `try_get(...).map(as_bool)` isn't sufficient? Or something like `Helper::get_bool(row, ...)`

Reply:
Yes, looks like it is best solution for now. Let leave it as is. 
(I don't like it much, because it looks like guessing and redundant job for processor. But I am new in Rust (3 monthes) and unable to propose something better for now... may be in the future :) )